### DISCUSSION TITLE
[Projects Beta] Display progression for issues with lists

### DISCUSSION THREAD

**Discussion by Sibz** [community/community#28635] [Projects and Issues]
Add a progression indicator to the board view for issues with lists.

Some of my project issues required a number of tasks to be complete, I have added then in a list, and the issue tracks the completion. But from the board view I have to drill into them to view the progress.

A progress indicator, perhaps something as simple is a mini pie chart would give progress information at a glance. This could be a opt-in feature in project settings.

**Comment by shouze** [comment]
Would love it a lot too! :heart_eyes:

**Comment by JonasWanke** [comment]
Looks like this is coming soon: https://github.com/orgs/community/discussions/16721#discussioncomment-2939031
